### PR TITLE
Add native math functions exp, pow

### PR DESCRIPTION
**Pull request by lankavitharana**

### CODE REVIEW

File: modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/math/Random.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+@BallerinaFunction(
+        packageName = "ballerina.lang.math",
+        functionName = "random",
+        returnType = {@ReturnType(type = TypeEnum.FLOAT)},
+        isPublic = true
+)
+public class Random extends AbstractNativeFunction {
+
+    public BValue[] execute(Context ctx) {
+        return getBValues(new BFloat(Math.random()));
```
Comment (sameera-jayasoma) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/math/Random.java]: Shall we add another native function to get a random number in a given range?

Comment (lankavitharana) [inline modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/lang/math/Random.java]: added method called "randomRange" which takes two int arguments and returns a int argument.

File: modules/ballerina-native/src/main/ballerina/ballerina/lang/math/natives.bal [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+@doc:Description { value:"Returns a random number between 0.0 and 1.0"}
+@doc:Return { value:"float: random value" }
+native function random () (float);
+
+@doc:Description { value:"Returns a random number between given start(inclusive) and end(exclusive) values"}
+@doc:Param { value:"start: range start value" }
+@doc:Param { value:"end: range end value" }
+@doc:Return { value:"float: random value" }
+native function randomRange (int start, int end) (int);
```
Comment (sameera-jayasoma) [inline modules/ballerina-native/src/main/ballerina/ballerina/lang/math/natives.bal]: We need to find a better name for this function :)

Comment (lankavitharana) [inline modules/ballerina-native/src/main/ballerina/ballerina/lang/math/natives.bal]: randomInRange?

**Pull request merged**